server/src/models/employer_contact.model.js

```diff
@@ -16,7 +17,7 @@ const EmployerContact = sequelize.define("employer_contacts", {
     type: DataTypes.STRING,
     allowNull: false,
   },
-  job_title: {
+  job_type: {
```

Comment:
is job type the same as employment type? is there some enum we can use to keep it consistent? 

e.g ` DataTypes.ENUM(["Full Time", "Part Time", "Casual", "On-Call"]),`

Reply:
This one is just a string. Job type refers to the position of the employee contact in the company.